Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```
@@ -762,17 +774,857 @@ script:
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.DisplayName
+      description: 'The display name of the eDiscovery search.'
+      type: String
+    - contextPath: MsGraph.eDiscoverySearch.LastModifiedDateTime
+      description: ''
+      type: Date
+    - contextPath: MsGraph.eDiscoverySearch.SearchId
+      description: 'The ID for the eDiscovery search.'
+      type: String
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: The id of the eDiscovery search.
+      isArray: false
+      name: search_id
+      required: true
+    - description: The display name of the search.
+      isArray: false
+      name: display_name
+      required: true
+    - description: The description of the search.
+      isArray: false
+      name: description
+      required: false
+    - description: The query string used for the search. The query string in KQL (Keyword Query Language) format. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
+      isArray: false
+      name: content_query
+      required: false
+    - description: When specified, the collection will span across a service for an entire workload.
+      isArray: false
+      name: data_source_scopes
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - none
+      - allTenantMailboxes
+      - allTenantSites
+      - allCaseCustodians
+      - allCaseNoncustodialDataSources
+    description: Update an eDiscoverySearch object.
+    execution: false
+    name: msg-update-ediscovery-search
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: The id of the eDiscovery search. If provided only this id will be returned.
+      isArray: false
+      name: search_id
+      required: false
+    - description: The max number of results to return. Default is 50.
+      isArray: false
+      name: limit
+      required: false
+      defaultValue: 50
+    - auto: PREDEFINED
+      description: Show all results if true.
+      isArray: false
+      name: all_results
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+    description: Get the list of eDiscoverySearch resources from an eDiscovery case.
+    execution: false
+    name: msg-list-ediscovery-searchs
+    outputs:
+    - contextPath: MsGraph.eDiscoverySearch.ContentQuery
+      description: 'The query string in KQL (Keyword Query Language) query. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.'
```

```suggestion
      description: 'The query string in KQL (Keyword Query Language) query. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values, for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.'
```

Where is this:  Keyword queries and search conditions for Content Search and eDiscovery?  Need a link or remove.